packages/expo-router/src/views/Sitemap.tsx

```diff
@@ -63,74 +61,34 @@ export function Sitemap() {
 }
 
 function FileSystemView() {
+  const sitemap = useSitemap();
   // This shouldn't occur, as the user should be on the tutorial screen
-  if (!store.routeNode) return null;
-  const children = store.routeNode.children.filter(({ internal }) => !internal).sort(sortRoutes);
-
-  return children.map((route) => (
-    <View testID="sitemap-item-container" key={route.contextKey} style={styles.itemContainer}>
-      <FileItem route={route} />
+  if (!sitemap) return null;
+  const children = sitemap.children.filter(({ isInternal }) => !isInternal);
+  return children.map((child) => (
+    <View testID="sitemap-item-container" key={child.contextKey} style={styles.itemContainer}>
+      <FileItem node={child} />
     </View>
   ));
 }
 
 function FileItem({
-  route,
+  node,
   level = 0,
-  parents = [],
   isInitial = false,
 }: {
-  route: RouteNode;
+  node: SitemapType;
   level?: number;
-  parents?: string[];
   isInitial?: boolean;
 }) {
-  const disabled = route.children.length > 0;
-
-  const segments = React.useMemo(
-    () => [...parents, ...route.route.split('/')],
-    [parents, route.route]
-  );
-
-  const href = React.useMemo(() => {
-    return (
-      '/' +
-      segments
-        .map((segment) => {
-          // add an extra layer of entropy to the url for deep dynamic routes
-          if (matchDeepDynamicRouteName(segment)) {
-            return segment + '/' + Date.now();
-          }
-          // index must be erased but groups can be preserved.
-          return segment === 'index' ? '' : segment;
-        })
-        .filter(Boolean)
-        .join('/')
-    );
-  }, [segments, route.route]);
-
-  const filename = React.useMemo(() => {
-    const segments = route.contextKey.split('/');
-    // join last two segments for layout routes
-    if (route.contextKey.match(/_layout\.[jt]sx?$/)) {
-      return segments[segments.length - 2] + '/' + segments[segments.length - 1];
-    }
-
-    const routeSegmentsCount = route.route.split('/').length;
-
-    // Join the segment count in reverse order
-    // This presents files without layout routes as children with all relevant segments.
-    return segments.slice(-routeSegmentsCount).join('/');
-  }, [route]);
-
-  const info = isInitial ? 'Initial' : route.generated ? 'Virtual' : '';
-
+  const disabled = node.children.length > 0;
+  const info = isInitial ? 'Initial' : node.isVirtual ? 'Virtual' : '';
```

Comment:
What do you think about changing this to:
```suggestion
  const info = isInitial ? 'Initial' : node.isGenerated ? 'Generated' : '';
```

I think that virtual was introduced, when sitemap was meant to be the representation of filesystem, rather then actual sitemap.

It's not exactly related to this PR, but to the Sitemap in general, so I am not sure if my comment should be part of this PR though.